components/atom/card/src/index.js

```diff
@@ -45,9 +45,13 @@ const AtomCard = ({
       onClick={redirectToHref}
       onKeyDown={redirectOnEnter}
     >
-      <div className={CLASS_MEDIA}>
-        <Media />
-      </div>
+      <>
```

Comment:
I think Fragment is not required in this case.

Reply:
Removed!